tests/test_subscriptionmanager.py

```diff
@@ -159,6 +159,44 @@ def host(_host):
         self.assertEqual(self.sm.logger.debug.call_count, 3)
         self.assertEqual(report.state, AbstractVirtReport.STATE_FINISHED)
 
+    @patch('rhsm.connection.UEPConnection')
+    def test_job_status_with_status_command(self, rhsmconnection):
+        rhsmconnection.return_value.has_capability.return_value = True
+        config = VirtConfigSection.from_dict({'type': 'libvirt', 'owner': 'owner'}, 'test', None)
+        report = StatusReport(config)
+        rhsmconnection.return_value.getJob.return_value = {
+            'state': 'RUNNING',
+        }
+        self.sm.check_report_state(report, status_call=True)
+        self.assertEqual(report.state, AbstractVirtReport.STATE_PROCESSING)
+        self.assertEqual(report._last_job_status, 'RUNNING')
+        def host(_host):
```

Comment:
Why do you use this method here? Wouldn't be enough to modify mocked data accordingly? Something like:

```
...
'updated': [ { 'uuid': '123'} ]
...
```

Reply:
Cut and pasted from another test. I changed it to your suggestion as the original was unnecessary.